JS/main.js

```diff
@@ -0,0 +1,86 @@
+if (!window.localStorage.getItem("products")) {
```

Comment:
In the initialProducts Array, because of that all products in this array are similar in some values, you can create a function that returns array of objects with for loop, and the id should be the iteration number **i**,, and in the price filed, you can generate a random number between two values, so here you can create this array with less code